public/app/features/admin/AdminEditUserCtrl.ts

```diff
@@ -1,19 +1,23 @@
 import _ from 'lodash';
+import { dateTime } from '@grafana/ui/src/utils/moment_wrapper';
```

Comment:
You don't need a full path to import `dataTime`, just import from `@grafana/ui` :)